### PR TITLE
Adds knight armour

### PR DESCRIPTION
**Pull request by doobiedoo23**
Deus vult, purge the heretic!

Adds three sets of medieval armour. The templar armour spawns in the chaplain's locker so you can start a new crusade.

Credit to KorPhaeron and CosmicScientist

Sprites:
https://camo.githubusercontent.com/5ab4956b0eeb60212b319fa881fe51bfd647774e/68747470733a2f2f692e696d6775722e636f6d2f6e535035705a582e706e67

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
???

**Comment by ghost** [comment]
does it protect you orr

**Comment by TDcoolguy300** [comment]
Check Der file changes Oprayx, it's not hard to follow

**Comment by TDcoolguy300** [comment]
I'll throw a DNM on here if you want to improve those sprites first

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -209,6 +209,21 @@
 	armor = list(melee = 30, bullet = 10, laser = 20,energy = 10, bomb = 20, bio = 0, rad = 0)
 	// Offer about the same protection as a hardhat.
 	flags_inv = HIDEEARS|HIDEEYES
+/obj/item/clothing/head/helmet/knight
+	name = "medieval helmet"
+	desc = "A classic metal helmet."
+	icon_state = "knight_red"
+	item_state = "knight_red"
+	armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)
```
Comment (danhibiki) [inline code/modules/clothing/head/helmet.dm:217]: people are going to raid the fuck out of chapel

**Comment by ghost** [comment]
something cool for the chaplain
yes +1 please add

**Comment by Chronitonity** [comment]
Its not like people use riot armor, i doubt they'll really go out of their way for that either

**Comment by danhibiki** [comment]
Theres a world of difference between breaking into the armory and breaking in the chapel. Or two windows.

**Comment by Chronitonity** [comment]
What im saying is people dont take the riot armor even when they can, if this slows people down like a hardsuit then its not a big deal.

**Comment by ghost** [comment]
It looks bad. Wait for the TG pr to update, pretty sure they will be getting some better sprites.

**Comment by Foosili** [comment]
Desperately needs new sprites

**Comment by TDcoolguy300** [comment]
I can safely close this one, the newer one has superior sprites
